Description: Changing the value type of http_connect_timeout

The value type of http_connect_timeout definition
is changed from Bool to Int value. Python treats
a value more than 1 as True but oslo config defines
Boolean values as 'true, '1', 'yes' and 'on'.
So http_connect_timeout is only configured 1 or None.

Change-Id: I53075cc04d0ccea543f8e657279534208ed03058
Closes-bug: #1368545.

Origin: upstream, https://review.openstack.org/#/c/126543
Bug-Ubuntu: https://bugs.launchpad.net/keystonemiddleware/+bug/1368545

Index: python-keystoneclient/keystoneclient/middleware/auth_token.py
===================================================================
--- python-keystoneclient.orig/keystoneclient/middleware/auth_token.py
+++ python-keystoneclient/keystoneclient/middleware/auth_token.py
@@ -235,7 +235,7 @@ opts = [
                 help='Do not handle authorization requests within the'
                 ' middleware, but delegate the authorization decision to'
                 ' downstream WSGI components'),
-    cfg.BoolOpt('http_connect_timeout',
+    cfg.IntOpt('http_connect_timeout',
                 default=None,
                 help='Request timeout value for communicating with Identity'
                 ' API server.'),
